src/bin/vip-dev-environment-create.js

```diff
@@ -15,7 +15,8 @@ import chalk from 'chalk';
  * Internal dependencies
  */
 import command from 'lib/cli/command';
-import { createEnvironment, printEnvironmentInfo, getEnvironmentName } from 'lib/dev-environment';
+import { createEnvironment, printEnvironmentInfo } from 'lib/dev-env/dev-env-core';
```

Comment:
I know it might be a bit longer to write, but what do you think about naming the folder and the files `dev-environment` instead of `dev-env`? I think it's a good option to maintain consistency and call it the same everywhere.

Reply:
Yeah I was on the fence with that one. I wanted to have it in the file name as well, so that would mean `lib/dev-environment/dev-environment-lando.js` for instance. I guess that is fine. Will change it ot that.